sharktuner/sharktuner/dispatch_parser.py

```diff
@@ -28,40 +30,115 @@ def parse_mlir(mlir_text: str, ctx: common.TunerContext) -> ir.Module:
     return mlir_module
 
 
+@dataclass
+class OpInfo:
+    # Name of parent func.FuncOp with "match_" prefix.
+    parent_function_name: str
```

Comment:
Can we remove this member and query the nearest parent of type funcopinterface instead?